bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,133 +87,200 @@ The following table shows the available configuration parameters for each thing.
+Feature availability is dependent on the version of hardware and firmware versions of your specific controller.
+
+#### Synchronize Time
+
+This configuration setting will instruct the binding to automatically update the controller's clock every 24 hours with the value from the openHab server.
+This is useful to keep the pool system clock set correct and automatically adjust for daylight savings time.
+
+| Channel Group                     | Channel   | Type   |    | Description                                              |
+| :------------------------------:  | :-------: | :----: | :-: | :------------------------------------------------------- |
+| pool, spa, aux[1-8], feature[1-8] | switch    | Switch | RW | Indicates the particulcar circuit or feature is on or off.  | 
+| "                                 | name      | String | R  | Name of circuit |
+| "                                 | feature   | String | R  | Feature of ciruit |
+| poolheat, spaheat                 | setpoint  | Number:Temperature | RW | Temperature setpoint |
+| "                                 | temperature | Number:Temperature | R | Current water temperature.  Note, the temperature is only valid while in either pool or spa mode. |
+| "                                 | heatmode  | String | R  | Heat mode configured.  Values: NONE, HEATER, SOLARPREFERRED, SOLAR |
+| schedule[1-9]                     | schedule  | String | RW | Summary string of schedule.  |
+| "                                 | type      | String | RW | Type of schedule.  Note, to actually write the program to the controller, this channel must be written to with the same value 2 times within 5s. Values: NONE, NORMAL, EGGTIMER, ONCE ONLY |
+| "                                 | start     | Number | RW | Time of day to start schedule expressed in minutes.  |
+| "                                 | end       | Number | RW | Time of day to end schedule expressed in minutes. In the case of EGG TIMER, this shoud be the duration. |
+| "                                 | circuit   | Number | RW | Circuit/Feature the schedule will control. |
+| "                                 | days      | String | RW | The days the schedule will run.  S=Sunday, M=Monday, T=Tuesday, W=Wednesday, R=Thursday, F=Friday, Y=Saturday |
+| status                            | lightmode | String | RW | Light mode. Values: OFF, ON, COLORSYNC, COLORSWIM, COLORSET, PARTY, ROMANCE, CARIBBEAN, AMERICAN, SUNSET, ROYAL, BLUE, GREEN, RED, WHITE, MAGENTA |
+| "                                 | solartemperature | Number:Temperature | R | Solar temperature sensor reading. |
+| "                                 | airtemperature   | Number:Temperature | R | Air temperature sensor reading. |
+| "                                 | uom              | String             | R | Unit of measure. Values: CELCIUS, FARENHEIT. |
```

Comment:
What's the purpose of this channel? The user can configure metric/imperial units globally in the openHAB UI. The framework takes care to convert them automatically, then.

Reply:
I agree, this can be removed.   Its just that the pentair controller can have different uom vs. the framework, but the conversion is done in the binding and this doesn't need to be brought out as a channel.